app/templatetags/elo.py

```diff
@@ -14,22 +14,23 @@
 
 @register.inclusion_tag("elo_change.html")
 def elo_change(value):
-    if value < -10:
-        icon = ELO_CHANGE_ICONS["double-down"]
-        color_class = "text-danger"
-    elif value < 0:
-        icon = ELO_CHANGE_ICONS["down"]
-        color_class = "text-danger"
-    elif value > 10:
-        icon = ELO_CHANGE_ICONS["double-up"]
-        color_class = "text-success"
-    elif value > 0:
-        icon = ELO_CHANGE_ICONS["up"]
-        color_class = "text-success"
-    else:
-        icon = ELO_CHANGE_ICONS["same"]
-        color_class = "text-muted"
+    if value is not None:
```

Comment:
Here instead of having a big if with a lot of code inside, we could short circuit and exit early.

`if value is None: return`

This helps make the code simpler (less nested `if`s)

Reply:
Another thing is that we still need to return the same dict here. `elo` should be `-`, `color_class` should be `text-muted` and the `icon` should be `same`.